### PR TITLE
Delete the cell-global VSchema path

### PR DESCRIPTION
**Pull request by davygeek**
When Delete keyspace in vtctld dashboard with recursive command like "DeleteKeyspace -recursive testorders" , if not delete VSchema info at the same time, The vtctld dashboard  will got an error.
Moreover, if you delete the logical keyspace, the VSchema info is not necessary to continue to retain.

### CODE REVIEW

**Comment by michael-berlin** [comment]
It looks like so far we expect the user to manually delete the VSchema. That's what we do in the unit test: https://github.com/youtube/vitess/blob/master/go/vt/topotools/rebuild_vschema_test.go#L143

Can you please delete that SaveVSchema call there?

I'll defer the actual review to Alain. He'll be back in two weeks.

**Comment by davygeek** [comment]
@michael-berlin thanks, I only add "SaveVSchema call " in this PR,   We can wait for Alain back to discuss.

File: go/vt/wrangler/keyspace.go [hunk 1]
```diff
@@ -897,6 +897,11 @@ func (wr *Wrangler) DeleteKeyspace(ctx context.Context, keyspace string, recursi
 		}
 	}
 
+	// Delete the cell-global VSchema path
+	// If not remove this, vtctld web page Dashboard will Display Error
+	vschema := &vschemapb.Keyspace{}
+	wr.ts.SaveVSchema(ctx, keyspace, vschema)
```
Comment (alainjobart) [inline go/vt/wrangler/keyspace.go]: You probably want to catch the error here:

if err := wr.ts.SaveVSchema(ctx, keyspace, vschema); err != nil && err != topo.ErrNoNode {
    return err
}

or something similar....

Comment (davygeek) [inline go/vt/wrangler/keyspace.go]: @michael-berlin  ok, thanks

**Comment by alainjobart** [comment]
Build error needs the import:
go/vt/wrangler/keyspace.go:902: undefined: vschemapb in vschemapb.Keyspace

Need to add this to import list:

vschemapb "github.com/youtube/vitess/go/vt/proto/vschema"

**Comment by alainjobart** [comment]
LGTM

**Comment by alainjobart** [comment]
@sougou merge as convenient.

**Pull request merged**